site/docs/remote-caching.md

```diff
@@ -52,6 +52,8 @@ The remote cache stores two types of data:
 * The action cache, which is a map of action hashes to action result metadata.
 * A content-addressable store (CAS) of output files.
 
+**Note:** The remote cache also stores the output of actions (i.e. the `stdout`/`stderr`); so when a cache hit occurs during a build, you may see the exact outputs as without having any cache hit. Keep in mind that just seeing expected std outputs doesn't mean it's really running, when using remote cache.
```

Comment:
I don't think we can universally say something like this here.  The choice of what each cache server emits to output is up to the server implementation.

My preference is to not make this comment, but I'll pass it around to other's who know more about the various remote cache implementations.

Reply:
I agree with @aiuto. I think what might be useful is to clarify what "action result metadata" in the previous sentence means.

`Action result metadata includes the action's stdout, stderr, paths of output files and exit code.`

WDYT @aiuto @emranbm ?

Reply:
> The choice of what each cache server emits to output is up to the server implementation
As we know, the cache servers are just simple HTTP servers required to just implement `PUT` and `GET` requests. So I think the implementation of the cache server doesn't matter, but the files Bazel actually `PUT`s.

Reply:
I like @emranbm 's comment more, because these docs are about how Bazel behaves with remote cache, not just remote caching in general. And Bazel does put the stdout/err in the cache, and display it when fetching from the cache, which may be misleading. Also, for servers that support remote execution in addition to remote caching, it's not up to the server implementation whether to put the stdout/err as part of the action results -- they are part of the results, it's not a choice.

Reply:
Thanks, for your reply.
Any decision made?

Reply:
Bowing out of decisions on this one, up to @buchgr .

Reply:
What I think could be improved from the original note is to remove the **bold** formatting as well as a link to the documentation on cache hit/miss. How about this statement?

Note that the remote cache additionally stores the stdout and stderr for every action. Inspecting the stdout/stderr of Bazel thus is not a good signal for [estimating cache hits](https://docs.bazel.build/versions/master/remote-caching-debug.html).

Reply:
@buchgr I just updated the PR with your suggestion. Check that, please.